src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,14 @@
-export const TodoList = () => {};
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Array<Todo>;
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(item => (
+      <TodoInfo key={item.id} todos={item} />
```

Comment:
```suggestion
      <TodoInfo key={item.id} todo={item} />
```
it is better to call 'todo', because you pass one element of todo to the props, not the whole array, which is probably why not all tests pass